core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -665,6 +665,19 @@ public NativeImageInvokerInfo build() {
                 //address https://github.com/quarkusio/quarkus-quickstarts/issues/993
                 nativeImageArgs.add("-J--add-opens=java.base/java.text=ALL-UNNAMED");
 
+                if (nativeConfig.enableReports) {
+                    if (graalVMVersion.isOlderThan(GraalVM.Version.VERSION_21_3_2)) {
+                        nativeImageArgs.add("-H:+PrintAnalysisCallTree");
+                    } else {
+                        nativeImageArgs.add("-H:PrintAnalysisCallTreeType=CSV");
```

Comment:
@zakkak can you confirm that having `-H:PrintAnalysisCallTreeType=CSV` triggers `-H:+PrintAnalysisCallTree` and we don't need to add it explicitly?

Reply:
Yes. It happens [here](https://github.com/oracle/graal/blob/59a4294796c260fbadc3e1fee957928d2fbf3d48/substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/reports/AnalysisReportsOptions.java#L43-L50).